client/pages/Puzzle1Page/Puzzle1Page.jsx

```diff
@@ -177,7 +187,9 @@ export default function Puzzle1Page() {
                     </Modal.Title>
                   </Modal.Header>
                   <Modal.Body>
-                    You have completed Puzzle 1!
+                    {stoppedTime > 0 && (
+                      <p>You have completed Puzzle 1 in {minutes}:{seconds?.toString().padStart(2, '0') || '00'}:{milliseconds?.toString().padStart(2, '0') || '00'}!</p>
```

Comment:
Probably a nitpick, but would look a little nicer in mm:ss.ss format rather than mm:ss:ss